controller:
  controller_serial: /dev/serial/by-id/usb-FTDI_Roboteq_MDC1460_port-bow-if00-port0
  load:
    retract_time_millis: 2000
    pause_time_millis:   250
    extend_time_millis:  500
  fire:
    extend_time_millis:  2000
    shoot_time_millis:   3000
